frontend/src/forms/manage-restaurant-form/MenuSection.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button } from '@/components/ui/button';
+import { FormDescription, FormField, FormItem } from '@/components/ui/form';
+import { useFieldArray, useFormContext } from 'react-hook-form';
+import MenuItemInput from './MenuItemInput';
+
+const MenuSection = () => {
+  const { control } = useFormContext();
+
+  const { fields, append, remove } = useFieldArray({
+    control,
+    name: 'menu',
+  });
+
+  return (
+    <div className="space-y-2">
+      <div>
+        <h2 className="text-2xl font-bold">Menu</h2>
+        <FormDescription>
+          Create your menu and give each time a name and price
+        </FormDescription>
+      </div>
+
+      <FormField
+        control={control}
+        name="menuItems"
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/30#issuecomment-2386843572>#1fc782</a><span> - 10/01/2024, 07:41 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Inconsistency between the field name used in useFieldArray and FormField. A similar issue was also found in frontend/src/forms/manage-restaurant-form/MenuSection.tsx (line 25).
</div>
<div id="fix">
<b>Fix:</b> Change the 'name' prop in the FormField component to match the name used in useFieldArray, which is 'menu'.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -23,7 +23,7 @@
    <FormField
      control={control}
 -    name="menuItems"
 +    name="menu"
      render={() => (
        <FormItem className="flex flex-col gap-2">
          {fields.map((item, index) => (
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged